dominion-solutions/laravel-mysqlite
# Fixed small bug with mysql_lpad where it would be null if given zeros…

# Details

Fixed small bug with mysql_lpad function where it would return null if any argument was given the value of 0 and added a test for it.

# Steps to test changes

1. Run "vendor/bin/phpunit --filter testLpadZero" to run the new unit test.
1. Make sure it asserts the correct value.
1. Run "vendor/bin/phpunit" to run all unit tests.
1. Make sure they all still function properly and nothing was broken.

# Issues Addressed

[CU-ccck4k](https://app.clickup.com/t/ccck4k) - Possible Bug in mod() function
